### PR TITLE
App Service beta-3.0: missed docs and tweaks

### PR DESCRIPTION
**Pull request by jackofallops**

### CODE REVIEW

File: website/docs/d/linux_web_app.html.markdown [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+page_title: "Azure Resource Manager: Data Source: azurerm_linux_web_app"
+description: |-
+  Gets information about an existing Linux Web App.
+---
+
+# Data Source: azurerm_linux_web_app
+
+Use this data source to access information about an existing Linux Web App.
+
+!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/f/main/website/docs/guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```
Comment (tombuildsstuff) [inline website/docs/d/linux_web_app.html.markdown:13]:
```suggestion
!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```

File: website/docs/d/linux_web_app.html.markdown [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+
+* `client_cert_mode` - The Client Certificate mode.
+
+* `connection_string` - A `connection_string` block as defined below.
+
+* `custom_domain_verification_id` - The identifier used by App Service to perform domain ownership verification via DNS TXT record.
+
+* `default_hostname` - The default hostname of the Linux Web App.
+
+* `enabled` - Is the TODO enabled?
```
Comment (tombuildsstuff) [inline website/docs/d/linux_web_app.html.markdown:62]: TODO?

File: website/docs/d/service_plan.html.markdown [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+page_title: "Azure Resource Manager: Data Source: azurerm_service_plan"
+description: |-
+  Gets information about an existing Service Plan.
+---
+
+# Data Source: azurerm_service_plan
+
+Use this data source to access information about an existing Service Plan.
+
+!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/f/main/website/docs/guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```
Comment (tombuildsstuff) [inline website/docs/d/service_plan.html.markdown:13]:
```suggestion
!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```

File: website/docs/d/windows_web_app.html.markdown [hunk 4]
```diff
@@ -0,0 +1,513 @@
...
+page_title: "Azure Resource Manager: Data Source: azurerm_windows_web_app"
+description: |-
+  Gets information about an existing Windows Web App.
+---
+
+# Data Source: azurerm_windows_web_app
+
+Use this data source to access information about an existing Windows Web App.
+
+!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/f/main/website/docs/guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```
Comment (tombuildsstuff) [inline website/docs/d/windows_web_app.html.markdown:13]:
```suggestion
!> **NOTE:** This is a 3.0 beta Data Source intended to provide an improved experience for Web Apps by splitting the older `azurerm_app_service` into O/S Specific variants, allowing better property validation and simpler configuration. Please see the [3.0 Beta guide](guides/3.0-beta.html.markdown) for more information on enabling and using this Data Source.
```

File: website/docs/d/service_plan.html.markdown (hunk 3 above)
Comment (tombuildsstuff) [reply website/docs/d/service_plan.html.markdown:13]: however this isn't OS Specific?

File: website/docs/d/windows_web_app.html.markdown [hunk 5]
```diff
@@ -0,0 +1,513 @@
...
+
+* `client_affinity_enabled` - Is Client Affinity enabled?
+
+* `client_cert_enabled` - Are Client Certificates enabled?
+
+* `client_cert_mode` - The Client Certificate mode.
+
+* `connection_string` - A `connection_string` block as defined below.
+
+* `custom_domain_verification_id` - The ID of the TODO.
```
Comment (tombuildsstuff) [inline website/docs/d/windows_web_app.html.markdown:56]: TODO

File: website/docs/d/windows_web_app.html.markdown [hunk 6]
```diff
@@ -0,0 +1,513 @@
...
+
+* `current_stack` - The Current Stack value of the Windows Web App.
+
+* `docker_container_name` - The name of the Docker Container in used.
+
+* `docker_container_registry` - The Container Registry where the Docker Container is pulled from.
+
+* `docker_container_tag` - The Docker Container Tag of the Container in use.
+
+* `dotnet_framework_version` - The version of .Net in use.
```
Comment (tombuildsstuff) [inline website/docs/d/windows_web_app.html.markdown:132]: Is this .net framework, .net (formally Core) or both?

File: website/docs/d/windows_web_app.html.markdown [hunk 7]
```diff
@@ -0,0 +1,513 @@
...
+* `share_name` - The Name of the File Share.
+
+* `type` - The Azure Storage Type.
+
+
+---
+
+A `trigger` block exports the following:
+
+* `private_memory_kb` - TODO.
```
Comment (tombuildsstuff) [inline website/docs/d/windows_web_app.html.markdown:471]: TODO

File: website/docs/r/linux_web_app.html.markdown [hunk 8]
```diff
@@ -153,7 +153,7 @@ A `application_stack` block supports the following:
 
 A `auth_settings` block supports the following:
 
-* `enabled` - (Required) Should the Authentication / Authorization feature is enabled for the Linux Web App be enabled?
+* `enabled` - (Required) Should the Authentication / Authorization feature is enabled for the Linux Web App?
```
Comment (tombuildsstuff) [inline website/docs/r/linux_web_app.html.markdown:156]:
```suggestion
* `enabled` - (Required) Should the Authentication / Authorization feature be enabled for the Linux Web App?
```

File: website/docs/r/linux_web_app.html.markdown [hunk 9]
```diff
@@ -301,7 +301,7 @@ A `http_logs` block supports the following:
 
 A `identity` block supports the following:
 
-* `type` - (Required) The ype of managed service identity. Possible values include: `ManagedServiceIdentityTypeSystemAssigned`, `ManagedServiceIdentityTypeUserAssigned`, and `ManagedServiceIdentityTypeSystemAssignedUserAssigned`.
+* `type` - (Required) The type of managed service identity. Possible values include: `ManagedServiceIdentityTypeSystemAssigned`, `ManagedServiceIdentityTypeUserAssigned`, and `ManagedServiceIdentityTypeSystemAssignedUserAssigned`.
```
Comment (tombuildsstuff) [inline website/docs/r/linux_web_app.html.markdown:304]:
```suggestion
* `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssigned, UserAssigned`.
```

**Review by tombuildsstuff — CHANGES_REQUESTED** [review]
A few comments inline but this otherwise LGTM

File: website/docs/r/linux_web_app.html.markdown [hunk 10]
```diff
@@ -301,7 +301,7 @@ A `http_logs` block supports the following:
 
 A `identity` block supports the following:
 
-* `type` - (Required) The type of managed service identity. Possible values include: `ManagedServiceIdentityTypeSystemAssigned`, `ManagedServiceIdentityTypeUserAssigned`, and `ManagedServiceIdentityTypeSystemAssignedUserAssigned`.
+* `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssignedUserAssigned`.
```
Comment (tombuildsstuff) [inline website/docs/r/linux_web_app.html.markdown:304]: the API defines these as:

```suggestion
* `type` - (Required) The type of managed service identity. Possible values include: `SystemAssigned`, `UserAssigned`, and `SystemAssigned, UserAssigned`.
```

File: website/docs/d/windows_web_app.html.markdown (hunk 6 above)
Comment (jackofallops) [reply website/docs/d/windows_web_app.html.markdown:132]: Both, the service handles this internally and refers to the setting as one thing. As discussed offline, I'll change this to `dotnet_version` to reduce the potential confusion.

**Review by katbyte — APPROVED** [review]
LGTM 🏗️

**Pull request merged**